DSharpPlus/Net/Rest/DiscordApiClient.cs

```diff
@@ -1785,6 +1785,26 @@ internal async Task<DiscordStageInstance> ModifyStageInstanceAsync(ulong channel
             return stage;
         }
 
+        internal async Task StageInstanceBecomeSpeakerAsync(ulong guildId, ulong id, DateTime? timestamp = null, bool suppress = false, ulong user = 0, string reason = "")
```

Comment:
This should be named more along the lines of `BecomeStageInstanceSpeakerAsync` I think?